popSimManu.tex

```diff
@@ -441,22 +454,19 @@ \subsection*{Use case: comparing methods of demographic inference}
 \citep{ragsdale2019models}, along with an empirical genetic map. In each column of this figure
 we show $N(t)$ inferred from samples taken from each of the three extant populations in the model.
 In each row we show comparisons among the methods (including two sample sizes for \MSMC).
-Blue lines show estimates from each of three replicate whole genome simulations.
-There is no single ``true'' reference for effective population size
-because of model misspecification---the inference methods are fitting a single population model
-to data simulated from multiple populations.
-However, many methods work by matching coalescence time distributions,
-and a single-population model with varying population size can match any coalescence time distribution
-(in which case coalescence rate is the inverse of the effective size).
-For this reason, we used as our ``ground-truth'' (solid black lines)
-not historical census sizes,
-but rather inverse coalescence rates calculated analytically in \texttt{msprime} (see Appendix).
+Blue lines show estimates from each of three replicate whole genome simulations and (FIXME: what color?)
```

Comment:
grey, maybe, so it doesn't overshadow the blue lines so much? or black but thinner?

Reply:
sounds good to me

Reply:
hmmm actually if its all the same, what if we do the coal rates in red and census in grey for consistency with the two pop figs (where census is grey)? tagging @jradrion since he volunteered to update the figs